metric/recorder.go

```diff
@@ -29,6 +30,12 @@ func recordReqsTotal(ctx context.Context, req *http.Request) {
 	reportRequestsTotal(labelsVal(ctx, req))
 }
 
+func recordResSize(ctx context.Context, req *http.Request) {
+	method, entity, code := labelsVal(ctx, req)
+	resSize := goa.ContextResponse(ctx).Length
```

Comment:
We are lucky here, we got Response_Size from goa.ResponseData only.  I verified and its giving correct size numbers.